### DISCUSSION TITLE
wake=2 for hero 7 black

### DISCUSSION THREAD

**Discussion by daich** [gopro/labs#405] [Feature Requests]
I'd love to have my hero 7 turn on when it receives usb power, no need to start recording.
The command wake=2 does just that but for some reason only works for hero 8 and above. Any particular reason the 7 isn't included?

**Comment by dnewman-gpsw** [comment]
That Labs feature came after the one and only release for HERO7.

**Reply by daich** [reply]
what is the possibility of getting an updated release for HERO7?

**Reply by dnewman-gpsw** [reply]
Future Labs updates are only likely H11/Mini and maybe H10.
